main.py

```diff
@@ -0,0 +1,63 @@
+# Task 1: Simple Quiz implementaion
+import json
+ 
+score = 0
+
+# Functions prototype
+def check_answer(entered_answer, correct_answer):
+    global score
+    if int(entered_answer) == correct_answer:        
+        score += 1  # Increase score for a correct answer
+        return "correct"
+       
+    else:
+        return "wrong"
+   
+ 
+def get_valid_choice():
+    while True:
+        try:
+            user_answer = int(input("Please, Enter the correct choice (1, 2, 3, 4): "))
+            if user_answer in [1, 2, 3, 4]:
+                return user_answer
+            else:
+                print("Invalid choice. Please enter a number between 1 and 4.")
+        except ValueError:
+            print("Invalid input. Please enter a number.")
+           
+               
+def main():  
+    global score  
+    # Reading file  
+    with open('data.json', 'r') as file:
+        data = json.load(file)
+    
+    num_of_options=1
+    questions_count = 0
+   
+    for question_type in ['sport', 'maths']:
```

Comment:
this can be replaced with 
`for question_type in data['quiz']:`
This is more dynamic